lib/ProductOpener/Display.pm

```diff
@@ -3898,6 +3898,14 @@ HTML
 						$user_template_data_ref->{edit_profile} = 1;
 						$user_template_data_ref->{orgid} = $orgid;
 					}
+					if (defined $User{pro_moderator}) {
+						my @org_members;
+						foreach my $member_id (keys %{$user_or_org_ref->{members}}) {
```

Comment:
Let's add a sort so that we always get users in the same order. In Perl keys %some_hash can return keys in different order every time it's executed.

```suggestion
						foreach my $member_id (sort keys %{$user_or_org_ref->{members}}) {
```